src/CookieStore.ts

```diff
@@ -186,6 +193,14 @@ Invalid value has been removed from localStorage to prevent subsequent failed pa
       }
     })
   }
+
+  /**
+   * Retrieve the global local storage if defined
+   * @returns Storage | undefined
+   */
+  private getLocalStorage(): Storage | undefined {
```

Comment:
I don't think we get any benefits of keeping this as a private method. We'd rather store it as a module-scoped variable, as `localStorage` is either defined or not (determined by the environment). Getter makes sense if the storage is eventually defined, but this is not the case.